src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/Commands/AddAzureRmDataLakeAnalyticsDataSource.cs

```diff
@@ -53,14 +53,6 @@ public class AddAzureDataLakeAnalyticsDataSource : DataLakeAnalyticsCmdletBase
         [ValidateNotNullOrEmpty]
         public string AccessKey { get; set; }
 
-        [Parameter(ValueFromPipelineByPropertyName = true, Position = 2, Mandatory = false,
-            ParameterSetName = DataLakeParameterSetName,
-            HelpMessage =
-                "Optionally indicates that this should now be the default storage account for the DataLakeAnalytics account."
-            )]
-        [ValidateNotNullOrEmpty]
-        public SwitchParameter Default { get; set; }
```

Comment:
@begoldsm why are you removing this parameter? This will result in a breaking change